QUARKUS-5611.md

```diff
@@ -0,0 +1,32 @@
+# QUARKUS-5611 Support Keycloak Dev Service when OIDC client is used without OIDC extension
+ 
+JIRA: https://issues.redhat.com/browse/QUARKUS-5611
+ 
+Support Keycloak Dev Service when OIDC client is used without OIDC extension
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/43609
+
+Upsteram issue:
+- https://github.com/quarkusio/quarkus/issues/43086
+
+## Scope of testing
+- Testing the Dev Service for `quarkus-oidc-client` extension without `quarkus-oidc`.
+- Tests will cover Dev Service user experience.
+- In addition there will be simple smoke tests to test basic oidc-client functionality in Dev Mode.
+- All test will use Keycloak as OIDC provider.
+
+## Existing test coverage
+- Partially covered using [`quarkus-extensions-combinations`](https://github.com/quarkus-qe/quarkus-extensions-combinations)
+
+## Automated test development
+- Test suite
+  - Adding new module similar to [security/keycloak](https://github.com/quarkus-qe/quarkus-test-suite/tree/main/security/keycloak).
```

Comment:
Compared to the test coverage in https://github.com/quarkusio/quarkus/pull/43609/files, what additional scenarios do you plan?

Reply:
The Dev Mode user experience will be similar
The main difference will be that I'll be checking only Dev Mode. The additional tests will be checking that `oidc-client` can get tokens from OIDC server.